29_configuring_opnsense_and_creating_vms_via_scripts_and_manual_10_nov_2022/README.md

```diff
@@ -0,0 +1,221 @@
+# Demo on: Setting up a production-like Kubernetes cluster for the first time, part 1, 10 Nov 2022
+
+
+### 1. In Proxmox [create pool k3s-simple1, install SDN and settup VLAN 10](https://github.com/Alliedium/devops-course-2022/blob/main/23_networks_vlan_nested_proxmox_cloud-init_27-oct-2022/practice.md).
```

Comment:
Please rename practice.md to README.md, refer not to a full URL, but just to a relative one  (../23-…/README.md), settup -> setup